Yes. The built failed on NixOS Master with enabled Apache2 with php7.0

`impure path /usr/local/lib used in link
collect2: error: ld returned 1 exit status
make: *** [Makefile:167: libphp7.la] Error 1
`

--- a/pkgs/development/libraries/openldap/default.nix
+++ b/pkgs/development/libraries/openldap/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, openssl, cyrus_sasl, db, groff }:
+{ stdenv, fetchurl, openssl, cyrus_sasl, db, groff, libtool }:
 
 stdenv.mkDerivation rec {
   name = "openldap-2.4.44";
@@ -13,11 +13,12 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  buildInputs = [ openssl cyrus_sasl db groff ];
+  buildInputs = [ openssl cyrus_sasl db groff libtool ];
 
   configureFlags =
     [ "--enable-overlays"
       "--disable-dependency-tracking"   # speeds up one-time build
+      "--enable-modules"
     ] ++ stdenv.lib.optional (openssl == null) "--without-tls"
       ++ stdenv.lib.optional (cyrus_sasl == null) "--without-cyrus-sasl"
       ++ stdenv.lib.optional stdenv.isFreeBSD "--with-pic";
@@ -34,6 +35,10 @@ stdenv.mkDerivation rec {
 
     rm -rf $out/var
     rm -r libraries/*/.libs
+  '';
+
+  postInstall = ''
+    chmod +x $out/lib/*.so
   '';
 
   meta = with stdenv.lib; {

